src/test/suite/fixtures/recorded/languages/typescript/clearFunk3.yml

```diff
@@ -0,0 +1,29 @@
+languageId: typescript
+command:
+  spokenForm: clear funk
+  version: 3
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: namedFunction}
+  usePrePhraseSnapshot: true
+  action: {name: clearAndSetSelection}
+initialState:
+  documentContents: |-
+    abstract class Foo {
+        protected abstract bar(): void;
+      }
+  selections:
+    - anchor: {line: 1, character: 26}
+      active: {line: 1, character: 26}
+  marks: {}
+finalState:
+  documentContents: |-
+    abstract class Foo {
+        ;
```

Comment:
Shall we use trailing `;` matcher here as well?

Reply:
sure